Sources/ArgumentParser/Parsing/Interactive.swift

```diff
@@ -9,6 +9,12 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if os(Linux)
+import Glibc
+#else
+import Darwin
+#endif
```

Comment:
Let's use `canImport` for these instead of `os` checks — you can see the other files that import `Glibc`.

Reply:
done